calico-enterprise_versioned_docs/version-3.16/getting-started/manifest-archive.mdx

```diff
@@ -27,9 +19,9 @@ This feature is:
 
 ## How to
 
-1. Go to the [Releases tab](https://docs.tigera.io/releases), and navigate to the appropriate release (3.0 and later).
+1. Download your patch release from the following table.
 
-1. In the left navigation, click [Release notes](../release-notes/index.mdx), and click the link, **Release archive**.
+   <ReleaseArchiveTable/>
 
 1. Untar the **release-x.x.x.tgz** to a local directory.
```

Comment:
I notice that the format for the file name does not match the current format.

Reply:
Changed it to `release-x.x.x-y.y.y.tgz`

Reply:
Maybe let's go with `release-vx.y.z-vx.y.z.tgz`. Does that seem less confusing to you?